semanticdb/semanticdb3/src/main/scala/scala/meta/internal/semanticdb3/Scala.scala

```diff
@@ -22,6 +23,31 @@ object Scala {
     def isEmptyPackage: Boolean = symbol == Symbols.EmptyPackage
     def isGlobal: Boolean = !isNone && Descriptor.descriptorLasts.contains(symbol.last)
     def isLocal: Boolean = !isNone && !isGlobal
+    def isMulti: Boolean = symbol.startsWith(";")
+    def flattenMulti: List[String] = {
```

Comment:
How about we rename this to `symbols` or `parts`?

Reply:
I agree this name is not ideal, it replaced a `def flatten(sym: Symbol): Symbol` in metac but `String#flatten` is already used by the scala-library. I would like to have "multi" somewhere in there, how about `asMulti: List[String]`?

Reply:
Sounds good.